web/package.json

```diff
@@ -133,7 +133,8 @@
     "verify": "VERIFY_NO_UPDATES=1 yarn update-world",
     "mockserver": "cd ../mockserver && node server.js",
     "prepare": "cd .. && husky install web/.husky",
-    "cypress-test": "cypress run",
+    "cypress-test": "TZ=Europe/Copenhagen cypress run",
+    "cypress-open": "TZ=Europe/Copenhagen cypress open",
```

Comment:
Hmm, could we avoid setting the environmental variable here?
It won't work on Windows and will make it harder for people to contribute.

Reply:
Hmm, I guess we could remove any tests related to time-sensitive data, but it's really annoying that it's not easier setting a specific timezone for Cypress :(
(See https://github.com/cypress-io/cypress/issues/1043)

Reply:
An alternative is mocking `new Date()` with https://docs.cypress.io/api/commands/clock, but I tried it without success and it also feels like this should be fixed on a higher level if possible :)

Reply:
I haven't really tried this at all yet but some quick searches led me to find this.

Maybe we can set the ENV variables like this:
```js
process.env['NODE_ENV'] = 'production';
```

It's completely backwards but it might work?

Reply:
I'll address this in a new PR to avoid messing up this one further 😄